src/main/java/webApp/QuestionRepository.java

```diff
@@ -8,5 +8,5 @@
  * A database for questions
  */
 @RepositoryRestResource(collectionResourceRel = "survey", path = "survey")
-public interface QuestionRepository extends PagingAndSortingRepository<Question, Long> {
+public interface QuestionRepository extends PagingAndSortingRepository<Question, Integer> {
```

Comment:
Will, there be an issue constantly going from int to Integer

Reply:
I don't think so. The conversion is automatic.